nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/nifi/util/hive/HiveJdbcCommon.java

```diff
@@ -402,6 +409,9 @@ public static long convertToCsvStream(final ResultSet rs, final OutputStream out
                             rowValues.add("");
                         }
                         break;
+                    case SQLXML:
+                        rowValues.add(StringEscapeUtils.escapeCsv(((java.sql.SQLXML) value).getString()));
```

Comment:
Shouldn't we take care of null `value` here, too?

Reply:
Yep missed that (nulls are handled earlier in the Avro code, but individually in the CSV code), will add both, good catch! I had tested with nulls but only for Avro output.